storagenode/trust/cache_test.go

```diff
@@ -0,0 +1,170 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package trust
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/zeebo/errs"
+)
+
+func TestCacheLoadCreatesDirectory(t *testing.T) {
+	dir, err := ioutil.TempDir("", "")
```

Comment:
We have a package `internal/testcontext` that handles temporary files, dirs, cleanup, timeouts etc.

Reply:
:+1: